war/src/main/js/pluginSetupWizardGui.js

```diff
@@ -461,9 +461,34 @@ var createPluginSetupWizard = function(appendTarget) {
 			setPanel(pluginSuccessPanel, { installingPlugins : installingPlugins, failedPlugins: true });
 			return;
 		}
-
+		
+		var attachScrollEvent = function() {
+			var $c = $('.install-console-scroll');
+			if (!$c.length) {
+				setTimeout(attachScrollEvent, 50);
+				return;
+			}
+			var events = $._data($c[0], "events");
+			if (!events || !events.scroll) {
+				$c.on('scroll', function() {
```

Comment:
hmm, correct me if I am wrong but 
```
var $c = $('.install-console-scroll');
...
$c.data('userScrolled', false);
```

counts as DOM manipulation but yeah.

The only thing that worries me that the the event can got fired a lot. 

...but by no means I want to veto this PR. It was more an observation.